### DISCUSSION TITLE
So how do I download this?

### DISCUSSION THREAD

**Discussion by OT2000** [Vortex2Oblivion/LeatherEngine-Extended-Support#16] [Q&A] [answered]
I already tried figuring out how to download the games using SC by searching it up online, and using VS Code. But I can’t figure it out.

**Comment by Vortex2Oblivion** [comment, marked as answer]
https://github.com/Vortex2Oblivion/LeatherEngine-Extended-Support/actions/workflows/workflow.yml
Scroll down and click on the one for your operating system 
Not the ones on the left scroll down

**Reply by OT2000** [reply]
thanks
